src/resources/cubemap.js

```diff
@@ -35,13 +35,13 @@ Object.assign(CubemapHandler.prototype, {
     patch: function (asset, registry) {
         this.loadAssets(asset, function (err, result) {
             if (err) {
+                // fire error event if patch failed
                 registry.fire('error', asset);
                 registry.fire('error:' + asset.id, err, asset);
                 asset.fire('error', asset);
             } else {
-                registry.fire('load', asset);
-                registry.fire('load:' + asset.id, asset);
-                asset.fire('load', asset);
+                // nothing to do since asset:change would have been raised if
+                // resources were changed.
```

Comment:
This looks odd. Maybe just `// no else clause required because...`?

Reply:
done